website/docs/plugin/framework/migrating/attributes-blocks/blocks-computed.mdx

```diff
@@ -0,0 +1,107 @@
+---
+page_title: 'Computed Blocks: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate computed blocks from SDKv2 to attribute validators in the plugin Framework.
+---
+
+# Blocks with Computed Fields
+
+Some providers, resources, and data sources include repeatable nested blocks in their attributes. Some blocks contain
+fields  with `Computed: true`, which means that the provider code can define the value or that it could come from the
+output of terraform apply (e.g., the ID of an EC2 instance). Refer to [Schemas - Blocks](/plugin/framework/FIXME) in the
+Framework documentation for details.
+
+This page explains how to migrate computed blocks from SDKv2 to the Framework. Refer to
+[Blocks](/plugin/framework/migrating/attributes-blocks/blocks) if you are looking for information about migrating blocks
+that do not contain computed fields.
+
+## SDKv2
+
+In SDKv2, blocks are defined by an attribute whose type is `TypeList`, or `TypeSet` and whose `Elem` field is set to a
+`schema.Resource` that contains a map of the block's attribute names to corresponding `schemaSchema` structs.
+
+```go
+map[string]*schema.Schema{
+	"example": {
+		Type:     schema.TypeList,
+		Computed: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"nested_example": {
+						Type:        schema.TypeString,
+						Computed:    true,
+						...
+```
+
+## Framework
+
+In the Framework, when working with protocol version 5, computed blocks are implemented using an attribute with a `Type`
+of `types.ListType` which has an `ElemType` of `types.ObjectType`.
```

Comment:
The example below shows usage of `Type` of `types.ListType` which has an `ElemType` of `types.ObjectType` for the migration of the `certificates` field in the `tls` provider from SDKv2 to the Framework. Or have I misunderstood?

Reply:
I guess I got a little confused here because the next sentence talks about protocol version 6 and nested attributes, then shows the List > Object schema. It might be good to move that code example above the protocol version 6 sentence and add a nested attributes example schema (or provide clarity on what the example code is showing with code comments)

Reply:
This looks good to me, thanks! We may want to show `ListNestedAttributes` in the example since `schema.TypeList` allows configuration references with list index syntax (we can off-handedly mention that `SingleNestedAttributes` is a good choice for single underlying objects, if they don't mind a breaking change but also dropping the awkward `[0]` in configuration references)

Reply:
Makes sense. Have updated.